projects/packages/jetpack-mu-wpcom/src/features/wpcom-simple-odyssey-stats/wpcom-simple-odyssey-stats.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Load the Odyssey stats feature on WordPress.com Simple Site.
+ * See https://github.com/Automattic/jetpack/tree/trunk/projects/packages/stats-admin
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+use Automattic\Jetpack\Stats_Admin\Dashboard as OdysseyStats;
+OdysseyStats::init();
```

Comment:
I think we may need to put this behind `wpcom_is_nav_redesign_enabled()` otherwise it'll add the Jetpack > "Stats App" menu item for all Simple sites.

Reply:
Ah yes that makes sense! Thanks for catching that